<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerUi</class>
 <widget class="QWidget" name="ServerUi">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Rockwell</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>ServerUi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	Background:rgb(23, 27, 31);
	color:rgb(255,255,255);
	
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="main_frame">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background:rgb(4, 26, 76);
	border:3px solid rgb(255, 255, 255);
	
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
		Background:rgb(152, 16, 57);
		border:none;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Close_Button">
         <property name="geometry">
          <rect>
           <x>457</x>
           <y>2</y>
           <width>35</width>
           <height>35</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
		background:rgb(152, 16, 57);
		color:rgb(0,0,0);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:0px solid rgb(255,255,255);
}

QPushButton::hover{
		background:rgb(23, 27, 31);
		color:rgb(57, 69, 81);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:0px solid rgb(255,255,255);

}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>2</y>
           <width>261</width>
           <height>38</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color:rgb(255,255,255);
}</string>
         </property>
         <property name="text">
          <string>ServerUi</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
        <widget class="QPushButton" name="minimizeUi_Button">
         <property name="geometry">
          <rect>
           <x>414</x>
           <y>2</y>
           <width>35</width>
           <height>35</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
		background:rgb(152, 16, 57);
		color:rgb(0,0,0);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:0px solid rgb(255,255,255);
}

QPushButton::hover{
		background:rgb(23, 27, 31);
		color:rgb(57, 69, 81);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:0px solid rgb(255,255,255);

}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QFrame{
		Background:rgb(69, 176, 230);
		border:none

}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
		Background:rgb(23, 27, 31);
		border:none

}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Centaur</family>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
		color:rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>SERVEUR MESSAGERIE QXORM</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="styleSheet">
            <string notr="true">QFrame{
		Background:rgb(23, 27, 31);
		border:none

}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QSpinBox" name="Port">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>14</y>
              <width>181</width>
              <height>38</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox{
		background:rgb(23,27,31);
		color:rgb(255,255,255);
		border:none;
		border-bottom:2px solid rgb(152, 16, 57);	
}</string>
            </property>
            <property name="minimum">
             <number>1024</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>63</x>
              <y>20</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Centaur</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color:rgb(255,255,255);
}</string>
            </property>
            <property name="text">
             <string>Port</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="Running_Button">
            <property name="geometry">
             <rect>
              <x>15</x>
              <y>100</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
		background:rgb(255,255,255);
		color:rgb(0,0,0);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:2px solid rgb(57, 69, 81);
}

QPushButton::hover{
		background:rgb(38, 39, 41);
		color:rgb(255,255,255);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:2px solid rgb(255,255,255);

}
</string>
            </property>
            <property name="text">
             <string>Run</string>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="StopServer_Button">
            <property name="geometry">
             <rect>
              <x>258</x>
              <y>100</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
		background:rgb(255,255,255);
		color:rgb(0,0,0);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:2px solid rgb(57, 69, 81);
}

QPushButton::hover{
		background:rgb(236, 49, 50);
		color:rgb(255,255,255);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:2px solid rgb(255,255,255);

}
</string>
            </property>
            <property name="text">
             <string>Quit</string>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="StopConnexion_Button">
            <property name="geometry">
             <rect>
              <x>135</x>
              <y>100</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
		background:rgb(255,255,255);
		color:rgb(0,0,0);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:2px solid rgb(57, 69, 81);
}

QPushButton::hover{
		background:rgb(38, 39, 41);
		color:rgb(255,255,255);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:2px solid rgb(255,255,255);

}
</string>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="Online_Button">
            <property name="geometry">
             <rect>
              <x>380</x>
              <y>100</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
		background:rgb(255,255,255);
		color:rgb(0,0,0);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:2px solid rgb(57, 69, 81);
}

QPushButton::hover{
		background:rgb(38, 39, 41);
		color:rgb(255,255,255);
		border-top-left-radius:5px;
		border-top-right-radius:5px;
		border-bottom-right-radius:5px;
		border-bottom-left-radius:5px;
		border:2px solid rgb(255,255,255);

}
</string>
            </property>
            <property name="text">
             <string>Online</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
